samples/nextjs/jss-create.js

```diff
@@ -43,6 +43,8 @@ module.exports = function createJssProject(argv, nextSteps) {
     createEmptyStarter();
   }
 
+  strip({ stripCode: argv.empty });
```

Comment:
Reminder to configure suffix as 'EMPTY' here (if you end up changing default to 'STRIP')

Reply:
Done